### DISCUSSION TITLE
支持隐藏“短视频”

### DISCUSSION THREAD

**Discussion by AFutureD** [Paladinfeng/MiaoProject#190] [Ideas]
# 描述

期望在“推荐”、“热门”、“排行”这三个页面内，将视频时长小于 5 分钟的视频进行隐藏。

# 原因

**MiaoProject** 或者说B站，在我个人的使用场景下，看得更多的是：
1. 番剧等影视内容
2. UP 主认真创作的视频内容

尤其不会用来看简单、模版化、AI 生成的、搬运的 等低质量内容。

而目前，B站在排行、推荐、热门等内容的推送上，类 “短视频” 内容日渐增加。

所以，希望 MiaoProject 能尽量提供一个“干净”的页面。

# 细节

## 隐藏范围

目前仅希望隐藏 “推荐”、“热门”、“排行” 这三个页面内的视频，而不隐藏 “关注页面” 内的视频。

## “短视频” 的定义

“短视频” 是我对于低质量的视频的一个简单称呼，包括但不限于：
1. AI 生成旁白的内容。如：电影解说。
2. 搬运的内容。如：搬运 youtube 某些热门内容。
3. 简单的内容。如：随意拼凑的搞笑视频。

目前，对于上述视频的鉴别较为困难，所以，以时长来简单过滤是一个低成本而有效的手段。

过滤 5分钟 以下的视频，在实际情况下，可以过滤超过50%的低质量内容， 同时也会过滤出不到 10% 左右的优质内容，但个人觉得，这是可接受的。因为对于用户比较在意的内容，理想的情况下会在关注列表出现，并且对于 B 站一个主打娱乐向的平台，漏看少量视频应该不算太严重的问题，毕竟因为“推荐” 而遗漏更优质的视频这个情况可能会更严重一些。

注意：以上定义仅个人意见。

**Comment by Paladinfeng** [comment]
隐藏区域增加：搜索
